### DISCUSSION TITLE
Esta permitido crear un grupo de teams para los explorers que se quieran juntar a compartir ideas?

### DISCUSSION THREAD

**Discussion by VictorFalconi** [Launch-X-Latam/MisionFrontEnd#157] [Q&A]
Estoy pensando crear un grupo de teams ya que es el medio por el cual vemos las clases y creo que nos beneficiaria bastante compartir ideas y conocimientos entre todos, asi podriamos llevarnos una mejor experiencia del curso. Por favor, algun moderador que me confirme si esta permitido!

**Comment by doguedogue** [comment]
Tienes razón 🚀#150

**Comment by gumodi35** [comment]
No esta mal la idea, pero como sugerencia no lo hagas en teams, en la 1era edición del Lauch y en esta también se menciono que no se podía hacer llamadas o grupos en el teams, ya lo mencionaron en algunas ocasiones, sin embargo hay algunas otras plataformas, el tema es lo del espacio seguro y las reglas de convivencia de la comunidad, es por eso que en teams no es posible, en cualquier otra plataforma es decisión y queda bajo la responsabilidad de quienes participan, mi consejo es que si la haces no la hagas en Teams. Por otro lado aquí se pueden ir siguiendo los hilos y resolviendo las dudas de la comunidad, te comento en base a lo que he visto en los programas de Launch X. Saludos¡

**Reply by VictorFalconi** [reply]
Bueno, lei por ahi la respuesta de una persona del staff que si se podia hacer grupo por teams ya que es otro medio de comunicacion oficial del curso pero pues necesito la confirmacion nuevamente por si fue un error o algo. Ojala me confirmen para poder hablar con los compañeros sin problemas

**Reply by jlcastrogro-launchx** [reply]
> No esta mal la idea, pero como sugerencia no lo hagas en teams, en la 1era edición del Lauch y en esta también se menciono que no se podía hacer llamadas o grupos en el teams, ya lo mencionaron en algunas ocasiones, sin embargo hay algunas otras plataformas, el tema es lo del espacio seguro y las reglas de convivencia de la comunidad, es por eso que en teams no es posible, en cualquier otra plataforma es decisión y queda bajo la responsabilidad de quienes participan, mi consejo es que si la haces no la hagas en Teams. Por otro lado aquí se pueden ir siguiendo los hilos y resolviendo las dudas de la comunidad, te comento en base a lo que he visto en los programas de Launch X. Saludos¡

No, eso está mucho pero. Usar otros medios está prohibido, lo dijeron en los primeros videos y lo siguieron repitiendo en otras discusiones.

**Comment by T0ny-dev** [comment]
Estaría bien , creo la moderadora ya comento que si se podía para solo platicar entre nosotros crear solo en teams , en otros medios no , pero si nos re-confirman estaría bien

**Reply by VictorFalconi** [reply]
Si, justo eso, quisiera una confirmacion directa para poder hacerlo y no incumplir normas
